tests/pod_queries_test.libsonnet

```diff
@@ -0,0 +1,54 @@
+local pod = import '../dashboards/resources/queries/pod.libsonnet';
+
+local config = {
+  _config: {},
+};
+
+local expectedWithRate = |||
+  sum by (k8s_pod_name) (
+    max by (k8s_cluster_name, k8s_namespace_name, k8s_pod_name) (
+      rate(k8s_pod_cpu_time_seconds_total{k8s_cluster_name=~'${cluster}', k8s_namespace_name=~'${namespace}', k8s_pod_name=~'${pod}'}[$__rate_interval])
+    )
+    /
+    max by(k8s_cluster_name, k8s_namespace_name, k8s_pod_name) (
+      k8s_container_cpu_request{k8s_cluster_name=~'${cluster}', k8s_namespace_name=~'${namespace}', k8s_pod_name=~'${pod}'}
+    )
+  )
+|||;
+
+local expectedWithoutRate = |||
+  sum by (k8s_pod_name) (
+    max by (k8s_cluster_name, k8s_namespace_name, k8s_pod_name) (
+      k8s_pod_memory_usage_bytes{k8s_cluster_name=~'${cluster}', k8s_namespace_name=~'${namespace}', k8s_pod_name=~'${pod}'}
+    )
+    /
+    max by(k8s_cluster_name, k8s_namespace_name, k8s_pod_name) (
+      k8s_container_memory_request_bytes{k8s_cluster_name=~'${cluster}', k8s_namespace_name=~'${namespace}', k8s_pod_name=~'${pod}'}
+    )
+  )
+|||;
+
+{
+  testRatioWithRate:
+    local result = pod.cpuUsageVsRequests(config);
+    assert result == expectedWithRate :
```

Comment:
TIL about `assert`.

Though, it might be more useful for the test suite to be based on `promtool` so that we can check PromQL syntax, and support recording and alert rule evaluation as well.

So these tests are fine, but I'd rather have an expansive `promtool` test suite, than a jsonnet test suite.